### PR TITLE
Feature#5 manage company branches

### LINKED ISSUE #5: Manage company branches (CRUD)
# REQUIREMENTS
* Create **CRUD** operations to manage company branches. 

# DESIGN

# Acceptance Criteria
- [ ] Admin can create a company branch.
- [ ] Admin can read  a company branch.
- [ ] Admin can  update a company branch.
- [ ] Admin can delete a company branch.
- [ ] Admin can see a list of company's branches.
- [ ] In the database must exist: name and address

### PR DESCRIPTION
**Pull request by EusebioAjas**
# Description
Create CRUD operations to manage company branches.
It include model, views and controller for company branches.

These changes complete the admin role and principal models( [issue #8](https://github.com/BC-MAR-21-ROR/kata-07-rejoj-checador-equipo-03-k07/issues/8)). 
See [issue #5](https://github.com/BC-MAR-21-ROR/kata-07-rejoj-checador-equipo-03-k07/issues/5) for more information.

 # Testing
> bundle install
> rails server
> rails db:setup
> navigate to: http://127.0.0.1:3000/admins/sign_in
> sign in with email: admin@example.com and password: password
> after a successful sign in, and perform CRUD operations.
## Screenshots 
 crud_company_branch

### CODE REVIEW

File: app/controllers/company_branches_controller.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+class CompanyBranchesController < ApplicationController
+  before_action :set_current_branch, only: %i[edit update destroy]
```
Comment (facundotoconas) [inline app/controllers/company_branches_controller.rb:2]: Why is this done?
so that the administrator only see their associated branches?

**Review by facundotoconas — COMMENTED** [review]
good screenshots!

**Review by facundotoconas — APPROVED** [review]
Good job!

**Review by anxhe — APPROVED** [review]

**Comment by anxhe** [comment]
Great job
🍻

**Pull request merged**